includes/Admin.php

```diff
@@ -191,4 +192,19 @@ public function prefill_post_title( $title ) {
 		// Otherwise it runs through wptexturize() and the like, which we want to avoid.
 		return isset( $post->post_title ) ? $post->post_title : '';
 	}
+
+	/**
+	 * Hook in before wp_ajax_inline-save and add KSES.
+	 *
+	 * @since 1.4.0
+	 *
+	 * @return void
+	 */
+	public function load_kses_before_inline_save() {
+		if ( ! isset( $_POST['post_type'] ) || Story_Post_Type::POST_TYPE_SLUG !== $_POST['post_type'] ) { // phpcs:ignore WordPress.Security.NonceVerification.Missing
+			return;
+		}
+		$kses = new KSES();
+		$kses->init();
```

Comment:
We usually do `$kses->remove_filters();` afterwards to clean things up nicely again. Can we do that here as well? Is there a matching hook after saving?

If not, do we need to hook into something other than `wp_ajax_inline-save`?

`save_post` + `wp_after_insert_post` seems fitting, but the latter is 5.6 only...

Reply:
The function [wp_ajax_inline_save](https://developer.wordpress.org/reference/functions/wp_ajax_inline_save/), calls `wp_die` after `edit_post`. I don't see the issue here, about keeping these hooked.